front/components/assistant/AssistantList.tsx

```diff
@@ -0,0 +1,65 @@
+import { AssistantPreview } from "@dust-tt/sparkle";
+import type {
+  ConversationType,
+  LightAgentConfigurationType,
+} from "@dust-tt/types";
+import { useRouter } from "next/router";
+import React from "react";
+
+interface AssistantListProps {
+  agents: LightAgentConfigurationType[];
+  handleAssistantClick: (
+    agent: LightAgentConfigurationType,
+    conversation?: ConversationType
+  ) => void;
+}
+
+const AssistantList: React.FC<AssistantListProps> = ({
```

Comment:
Let's be consistent and use a regular function here.

Reply:
Plus it has been removed: https://github.com/facebook/create-react-app/pull/8177.

Reply:
:sweat_smile: forgot to review & clean this new file (imported from Thomas' initial PR)
Sorry about that & thx for the review